### ISSUE TITLE
<cv-file-uploader> Failed to resolve import

### ISSUE THREAD

**Issue by krko12345** [carbon-design-system/carbon-components-vue#1573]
Failed to resolve import

## Detailed description

> Describe in detail the issue you're having.

CvFileUploader throws error  when used in my project. The error is "Failed to resolve import", which is related to CvForm component import in the file-uploader component.

In my env.d.ts  I have declared the path as "declare module '@carbon/vue/src/components/*'", and I tried with different variations as well.

Still getting error:
![image](https://github.com/carbon-design-system/carbon-components-vue/assets/36237241/58ef5168-64f3-4fd6-b307-367899c5142f)

What works for me is to modify the CvFileUploader component and remove the CvForm wrapper inside, but of course that's not permanent solution.

Any recommendation? Is this a bug?

> Is this a feature request (new component, new icon), a bug, or a general issue?
issue

> Is this issue related to a specific component?
CvFileUploader

> What did you expect to happen? What happened instead? What would you like to see changed?
Expected the component to load, instead it throws error.

> What browser are you working in?
Edge

> What version of the Carbon Design System are you using?
3.0.12

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

- Screenshots or code
- Notes

**Comment by krko12345** [comment]
The problem seems to move to next line if I modify it to use parent directory path (instead of current directory) path for the import

![image](https://github.com/carbon-design-system/carbon-components-vue/assets/36237241/d5ed20e6-0c5c-4751-b607-0660c92fe453)

**Comment by davidnixon** [comment]
Hmm I cannot reproduce this. I think you are correct that its may have something to do with 'env.d.ts' file. What happens if you remove the "declare module " statement?

**Comment by krko12345** [comment]
Error remained even after removal of all declarations from "env.d.ts". 
It wouldn't be a solution anyway because without declarations the code would fail to compile from typescript.

**Comment by krko12345** [comment]
Not sure what more should I include in my response. The problem still persists as described.

I only managed to avoid the problem by having a modified copy of CvFileUploader component, where I have removed the CvForm wrapper. As mentioned, other carbon components work for me as expected.

**Comment by davidnixon** [comment]
I do not think I fully understand the use case but I think I do see the issue. Several index files import like this:
```js
import CvForm from './CvForm';
```
Notice the lack of ".vue" extension! 
I updated those places to import like this:
```js
import CvForm from './CvForm.vue';
```
 Could you try this patch?
[carbon-vue-3.0.16-5f775.tgz](https://github.com/user-attachments/files/15890241/carbon-vue-3.0.16-5f775.tgz)

```sh
npm remove @carbon/vue 
npm add ./carbon-vue-3.0.16-5f775.tgz
```

Another possible fix. Can you remove "declare module '@carbon/vue/src/components/*'"? I think maybe your use case is related to an issue @OlkaB resolved in April. #1578 which nows allow you to import individual components. So this should work without the `declare module`.
```js
import { CvFileUploader } from '@carbon/vue';
```

**Comment by krko12345** [comment]
@davidnixon  I confirm, the CvFileUploader component worked as expected while using carbon-vue-3.0.16-5f775.tgz

**Comment by davidnixon** [comment]
Fixed in #1606
